### PR TITLE
Add Affinity, AGAP2, BOLD, Checkmarx, PHC

### PR DESCRIPTION
**Pull request by antoniomapinho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +49,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [Checkmarx](https://www.checkmarx.com/) [:rocket:](https://www.checkmarx.com/company/careers/) | "Manage Software Exposure at the Speed of DevOps" | `Braga` |
```
Comment (marmelo) [inline README.md]: Please remove the `"` quotes from the description and end it with a `.` period.

File: README.md [hunk 2]
```diff
@@ -74,6 +79,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | :------ | :---------- | :-------- |
 | [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://www.criticalmanufacturing.com/en/careers) | Manufacturing business-critical applications. | `Porto` |
 | [Hitachi Vantara](https://www.hitachivantara.com/) [:rocket:](https://www.hitachivantara.com/en-us/company/careers.html) | IoT, cloud, application, big data and analytics solutions. | `Oeiras` `Porto` |
+| [PHC Software](http://www.phcsoftware.com/) [:rocket:](https://www.phcsoftware.com/carreiras/) | ERP software | `Lisbon` `Porto` |
```
Comment (marmelo) [inline README.md]: Please end the description with a . period.

**Comment by marmelo** [comment]
Hi @antoniomapinho, can you please rebase and check my review comments? Thanks!

**Comment by antoniomapinho** [comment]
Thanks. Rebased and pushed changes.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```
Comment (marmelo) [inline .gitignore]: Can you please remove this file? Thanks.

File: README.md [hunk 4]
```diff
@@ -23,8 +23,11 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
 | [AddCode](https://addcode-io.breezy.hr) [:rocket:](https://addcode-io.breezy.hr/#positions) | Software engineering. | `Porto` |
+| [Affinity](https://affinity.pt) [:rocket:](https://affinity.pt/en/careers/) | Consultancy, Outsourcing. | `Lisbon` `Porto` |
```
Comment (marmelo) [inline README.md]: Can you please replace `Lisbon` with `Lisboa`?  
This also applies to the other entries.
Thanks!

File: README.md [hunk 5]
```diff
@@ -75,6 +80,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | :------ | :---------- | :-------- |
 | [Critical Manufacturing](https://www.criticalmanufacturing.com) [:rocket:](https://www.criticalmanufacturing.com/en/careers) | Manufacturing business-critical applications. | `Porto` |
 | [Hitachi Vantara](https://www.hitachivantara.com/) [:rocket:](https://www.hitachivantara.com/en-us/company/careers.html) | IoT, cloud, application, big data and analytics solutions. | `Oeiras` `Porto` |
+| [PHC Software](http://www.phcsoftware.com/) [:rocket:](https://www.phcsoftware.com/carreiras/) | ERP software. | `Lisboa` `Porto` |
```
Comment (marmelo) [inline README.md]: Travis is reporting that the following link is return a `HTTP 500` error:
http://www.phcsoftware.com/

I have checked and the same one with HTTPS seems to work OK.  
Can you please check and update it?

Thanks.

Comment (antoniomapinho) [reply README.md]: Done. I had actually hit it with https but typed it wrong.

**Comment by marmelo** [comment]
It took some effort but it is done!  
Thanks @antoniomapinho! :star:

**Comment by antoniomapinho** [comment]
Sorry, unwisely decided to contribute parallel to resolving a bug lead to me not paying due attention.

**Comment by marmelo** [comment]
No problem, it happens to all of us @antoniomapinho!

**Pull request merged**